### PR TITLE
fix(build-test): Fix failing RN build test

### PR DESCRIPTION
**Pull request by esauerbo**
#### Checklist

- [ ] Have read the [Pull Request Guidelines](https://github.com/aws-amplify/amplify-ui/blob/main/CONTRIBUTING.md)
- [ ] PR description included
- [ ] `yarn test` passes and tests are updated/added
- [ ] PR title and commit messages follow [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/#summary) syntax

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: build-system-tests/package.json [hunk 1]
```diff
@@ -21,7 +21,7 @@
     "test:vue:nuxt": "./scripts/setup-mega-app.sh -F vue --build-tool nuxt",
     "setup:react-native": "./scripts/setup-mega-app.sh -F react-native",
     "setup:react-native:cli": "./scripts/setup-mega-app.sh -F react-native -B cli",
-    "test:react-native:expo": "./scripts/setup-mega-app.sh -F react-native -B expo",
+    "setup:react-native:expo": "./scripts/setup-mega-app.sh -F react-native -B expo",
```
Comment (jordanvn) [inline build-system-tests/package.json:24]: Info: if this is running the setup command, do you why was it labeled as "test"?

Comment (esauerbo) [reply build-system-tests/package.json:24]: I renamed to test in https://github.com/aws-amplify/amplify-ui/pull/5878 but doing web and RN separately so trying to revert this for now. `setup-mega-app` runs the whole build test suite so it's `test` is more accurate

Comment (jordanvn) [reply build-system-tests/package.json:24]: Ah, ok. So are we going to rename it to `test` again in the future?

**Review by jordanvn — APPROVED** [review]

File: build-system-tests/package.json (hunk 1 above)
Comment (esauerbo) [reply build-system-tests/package.json:24]: Yes

**Review by reesscot — APPROVED** [review]

**Pull request merged**